apps/ledger-live-desktop/src/renderer/screens/earn/useEarnStakeProgramsParam.ts

```diff
@@ -0,0 +1,18 @@
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+
+export function useEarnStakeProgramsParam(): Record<string, string> | undefined {
+  const stakePrograms = useFeature("stakePrograms");
+
+  console.log("stakePrograms", stakePrograms);
+  const redirects = stakePrograms?.params?.redirects ?? {};
+  const returns: Record<string, string> = {};
+  const keys = Object.keys(redirects);
+  if (keys.length === 0) {
+    return undefined;
+  }
+  keys.forEach(key => {
+    return (returns[key] = redirects[key].platform);
```

Comment:
The `return` is not needed here and a bit confusing IMO:
```suggestion
    returns[key] = redirects[key].platform;
```